sympy/stats/crv_types.py

```diff
@@ -922,7 +922,7 @@ def Exponential(name, rate):
     Parameters
     ==========
 
-    rate : A positive Real number, `\lambda > 0`, the rate (or inverse scale/inverse mean)
+    l : A positive Real number, `\lambda > 0`, the rate (or inverse scale/inverse mean)
```

Comment:
This should be the same as in the function definition of `Exponential` and in `_argnames` of `ExponentialDistribution`.

Reply:
I have restored it back to `rate`